### DISCUSSION TITLE
Multitasking/screenshot privacy settings

### DISCUSSION THREAD

**Discussion by Donkey-Doug** [standardnotes/forum#2833] [General]
Changes made to the disable multitasking/screenshot privacy setting are not saved. I cannot disable this option. Every restart it is turned on again.

Android 10
Version 3.23.8

**Comment by myreli** [comment]
Hi, @Donkey-Doug does this still happen with the latest version? Since the new mobile experience, I think it's been fixed, but my Android version is different.
